jupyterlab_git/git.py

```diff
@@ -812,15 +801,17 @@ def get_current_branch(self, current_path):
         )
         output, error = p.communicate()
         if p.returncode == 0:
-            return output.decode("utf-8").strip()
+            return output.decode("utf-8").split('/')[-1].strip()
+        elif "not a symbolic ref" in error.decode("utf-8"):
```

Comment:
Should we compare lower case error message to prevent error seen elsewhere #409:

```suggestion
        elif "not a symbolic ref" in error.decode("utf-8").lower():
```